app/src/main/java/com/pajato/android/gamechat/chat/ChatManager.java

```diff
@@ -74,6 +74,23 @@ public boolean startNextFragment(final FragmentActivity context) {
         return dispatcher.type != null && startNextFragment(context, dispatcher);
     }
 
+    /** Directly navigate to the Me Room. Return true iff the fragment is started. */
```

Comment:
I'm not crazy about adding new methods to handle these items.  Better to use existing methods and tailor the arguments.